health/micro-ui/web/micro-ui-internals/packages/modules/health-dss/src/utils/getDuration.js

```
@@ -0,0 +1,12 @@
+export const getDuration = (startDate, endDate) => {
+  let noOfDays = (new Date(endDate).getTime() - new Date(startDate).getTime()) / (1000 * 3600 * 24);
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'noOfDays' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)